apps/daimo-mobile/src/sync/sync.ts

```diff
@@ -25,11 +25,11 @@ import { Account, getAccountManager } from "../model/account";
 export function startSync() {
   console.log("[SYNC] APP LOAD, starting sync");
   maybeSync(true)
-    .then((success) => {
-      if (!success) {
+    .then((status) => {
+      if (status === "failed") {
         updateNetworkState(() => {
-          // set fail to 3 because at 2 it would remove offline banner
-          // for one try even if sync failed
+          // HACK: mark offline if splash screen sync fails due to network state
+          // by setting syncAttemptsFailed to 3
```

Comment:
dont think the hack should be necessary--can check, but just

`{ status: "offline", syncAttemptsFailed: 1 }` should do it imo

Reply:
done, this was old code -- I think it was trying to make the first load show offline by itself, but we don't need that, fair.